src/components/page-header/page-header.component.less

```diff
@@ -141,12 +142,17 @@ ux-page-header {
 
             .page-header-condensed-content {
                 position: relative;
+                width: 100%;
```

Comment:
Are these changes related to the new template options, or a separate issue? If the latter, please revert and open a ticket.

Reply:
These changes are relating to the main header not centering when condensed is set to true. They are unrelated to the ticket but worth fixing so I thought I'd do it in the same ticket. I'll remove them and add a new ticket in the backlog.

Reply:
Actually I also found the alignment on secondary nav is not working in the current release. So on second thoughts I think these changes are needed. The alignment will be skewed if they are not in when both navbars are center-aligned so I think it is best to keep them. @donna-marijne what do you think?

Reply:
Secondary nav alignment looks fine to me. Is this the change that broke the position of the icon menu?

Reply:
No the issue you mentioned is in the quantum-ux-aspects CSS, not the UxAspects open source. This is an issue with alignment in UxAspects - the alignment for secondary nav doesn't work which impacts the masthead as it uses the page-header. I've resolved the alignment issues in these changes. See https://plnkr.co/edit/BWpjPM6VEmoL1Bdx

Reply:
As mentioned previously, secondary navigation is specific to the Micro Focus theme, which is why it is not documented in this repository. Please don't waste time on this.